docs/plugin/troubleshooting.md

```diff
@@ -0,0 +1,89 @@
+---
+id: troubleshooting.md
+title: "Troubleshooting"
+sidebar_position: 6
+---
+
+This document describes how to identify and resolve issues that might occur.
+
+### You don't receive source events
+
+**Symptoms**
+
+You don't get event message on a given communication platform even though the event occurred in your 3rd party service.
+
+**Debugging steps**
+
+- [Enable debugging mode for a given plugin](debug.md).
+  Once enabled, check the Botkube logs. You can filter them by your plugin name. You should see information about downloading and starting your source. For example:
+
+  ```bash
+  INFO[2023-01-09T21:21:24+01:00] Starting Plugin Manager for all enabled plugins  component="Plugin Manager" enabledSources=botkube/cm-watcher
+  INFO[2023-01-09T21:21:24+01:00] Downloading plugin.                           binPath=/tmp/plugins/botkube/source_v0.13.0_cm-watcher component="Plugin Manager" url="http://host.k3d.internal:3000/static/source_cm-watcher_darwin_amd64"
+  INFO[2023-01-09T21:21:24+01:00] source plugin registered successfully.        binPath=/tmp/plugins/botkube/source_v0.13.0_cm-watcher component="Plugin Manager" plugin=botkube/cm-watcher version=v0.13.0
+  INFO[2023-01-09T21:21:25+01:00] Start source streaming...                     pluginName=botkube/cm-watcher sources="[plugin-based]"
+  ```
+
+  If you don't see any of the above log messages:
+
+  - Make sure that `source.{group_name}.{plugin_name}.enabled` property is set to `true`
+  - Make sure that a given source configuration (`sources.{group_name}`) is bind to a given communication platform (`bindings.sources: [{group_name}]`)
+
+  If the source is not bound to any communication platform Botkube will not download and start such plugin. Even if it's enabled.
+
+- [Make sure that your plugin didn't crash](#plugin-process-exited).
+
+### You don't receive executor response
```

Comment:
This header starting with "you" sounds a bit odd to me. Maybe we can change it to:
```suggestion
### Missing executor response
```
or something similar.
That will be consistent with other headers (e.g. Plugin not found error) 🙂